docs/content/preview/deploy/manual-deployment/system-config.md

```diff
@@ -35,6 +37,31 @@ As of CentOS 8, `ntp` is no longer available and has been replaced by `chrony`.
 $ sudo yum install -y chrony
 ```
 
+### PHC (PTP Hardware Clock) {{<tags/feature/tp>}}
+
+PTP Hardware Clock (PHC) refers to a physical clock device that supports the [Precision Time Protocol (PTP)](https://en.wikipedia.org/wiki/Precision_Time_Protocol) which is a network protocol used for highly accurate time synchronization.
+
+To check if your AWS instance supports PHC, see [AWS PTP Hardware Clock](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configure-ec2-ntp.html#connect-to-the-ptp-hardware-clock).
+
+Configure PTP using
+
+```sh
+sudo bash ./bin/configure_ptp.sh
+```
+
+### Clockbound Agent {{<tags/feature/tp>}}
+
+Configuring a clockbound agent improves clock accuracy significantly.
```

Comment:
> so PHC is optional for any installation? for ntp only?
1. The above recommendation to install the ntp package is inaccurate. Users must install chrony whenever possible (and CentOS7 is End-Of-Life anyway).
2. You can configure clockbound with or without PHC.

> chrony is prerequisite for clockbound agent only?
However, you need chrony for configuring clockbound. So, this is a pre-requisite for clockbound.


Added a small phrase saying

pre-requisite for configuring clockbound.

Reply:
Should we just change the ntp section to chrony only?
Do PHC and clockbound belong as subsections of ntp/chrony, or should they be at the same level?

Reply:
> Should we just change the ntp section to chrony only?

Done.
Additional context: https://shorturl.at/dIdwt strongly recommends using chrony for example.

> Do PHC and clockbound belong as subsections of ntp/chrony, or should they be at the same level?

This is very subjective. What prompts you to ask this question? I prefer to keep this under ntp/chrony since PHC and clockbound are an extension of time synchronization. In fact, I would've named this section as time sync. Time sync is strictly necessary for distributed transactions which is at the heart of YugabyteDB.